docker-compose.yml

```diff
@@ -23,6 +23,11 @@ services:
       - --save ""
       - --appendonly no
 
+  rabbitmq:
+    image: rabbitmq:alpine
+    # ports:
+    #  - "5555:5555"
```

Comment:
Shall we specify the ports here?

Reply:
No, as I don't think we should expose them

Reply:
Removed